RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRecycleBin.java

```diff
@@ -498,16 +498,18 @@ private AbstractFile getOrMakeFolder(SleuthkitCase skCase, FsContent dataSource,
      * @throws TskCoreException
      */
     private void addFileSystemFile(SleuthkitCase skCase, FsContent recycleBinFile, Content parentDir, String fileName, long deletedTime) throws TskCoreException {
-        skCase.addFileSystemFile(recycleBinFile.getFileSystemId(),
+        skCase.addFileSystemFile(
                 recycleBinFile.getDataSourceObjectId(),
+                recycleBinFile.getFileSystemId(),
+                fileName,
+                recycleBinFile.getMetaAddr(),
+                (int) recycleBinFile.getMetaSeq(),
                 recycleBinFile.getAttrType(),
                 recycleBinFile.getAttributeId(),
-                fileName,
                 TskData.TSK_FS_NAME_FLAG_ENUM.UNALLOC,
+                (short) (TskData.TSK_FS_META_FLAG_ENUM.UNALLOC.getValue() | TskData.TSK_FS_META_FLAG_ENUM.USED.getValue()),
                 recycleBinFile.getSize(),
                 0, 0, 0, deletedTime,
```

Comment:
Can you add in the Timestamps from the $R file when you add it back in.  This keeps the file consistent with the $R file.

Reply:
This is not as straight forward as you might think.  Those values can only be added for only an $R that represents a deleted file.  Not all files.

Reply:
Fixed.